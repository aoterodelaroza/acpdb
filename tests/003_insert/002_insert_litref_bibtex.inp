## check: 002_insert_litref_bibtex.out -a1e-10
## delete: 002_insert_litref_bibtex.db
## labels: regression quick

verbose
system rm -f 002_insert_litref_bibtex.db
connect 002_insert_litref_bibtex.db
insert litref bibtex ../zz_source/bsip2.bib
print

