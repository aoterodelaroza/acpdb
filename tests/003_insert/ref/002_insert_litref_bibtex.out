* SYSTEM: rm -f 002_insert_litref_bibtex.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 002_insert_litref_bibtex.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)

* PRINT: print the contents of the database 

# Table of property types
| id | key | description |
| 1 | energy_difference | A difference of molecular or crystal energies (reaction energy, binding energy, lattice energy, etc.) |
| 2 | energy | The total energy of a molecule or crystal |

# Number of literature references: 234

# Table of methods
| id| key| gaussian_keyword| psi4_keyword| litrefs| description|

# Table of sets
| id| key| property_type| litrefs| description|

# Number of properties and structures in each set
| id | key | properties | structures |

## Description of the training set
# The TRAINING SET is NOT DEFINED
--- No atoms found (ATOM) ---
--- No angular momenta found (LMAX) ---
--- No exponents found (EXP) ---
--- No subsets found (SUBSET) ---
--- No weights found (W) ---
--- No empty method found (EMPTY) ---
--- No reference method found (REFERENCE) ---

## Table of saved training sets in the database
| Name |

